bin/s3rver.js

```diff
@@ -63,3 +63,17 @@ new S3rver(program).run(function(err, host, port) {
     console.log("now listening on host %s and port %d", host, port);
   }
 });
+
+s3rver.s3Event.subscribe(function(event) {
+  // eslint-disable-next-line no-console
+  console.log(event);
```

Comment:
@anantab can we disable this logging based on the `silent` CLI option? Would prefer to make logging configurable instead of forcing users to see noisy logs.

Alternatively you could add another CLI option to turn on/off S3 event logging independently of the internal logger.